docs/contributors/check-accessibility.md

```diff
@@ -0,0 +1,179 @@
+
+All graphical interfaces in Ubuntu Desktop should be readable with the built-in [Orca screen reader](https://canonical-ubuntu-desktop-documentation.readthedocs-hosted.com/en/latest/how-to/accessibility/read-screen-aloud/). Screen readers benefit blind and visually impaired users who rely on audio feedback to navigate interfaces. To work properly, screen readers require good markup, labelling, and logical content structure.
+
+### Use the screen reader
+
+The Orca screen reader can be enabled in {menuselection}`Settings --> Accessibility --> Seeing`. You can also enable {guilabel}`Screen Reader` in the Accessibility menu.
+
+Note that some Orca features don’t work with certain combinations of Ubuntu releases and application toolkits. Read [Improve screen reader usability](https://canonical-ubuntu-desktop-documentation.readthedocs-hosted.com/en/latest/how-to/accessibility/improve-screen-reader-usability/).
+
+You need to get familiar with keyboard shortcuts to use the screen reader. Modifier keys are used in the shortcuts:
+
+{kbd}`Super`
+: Generally mapped to the {kbd}`Windows` key on Windows computers, and the {kbd}`Command` key on Macs.
+
+{kbd}`Orca`
+: By default, the Orca key is {kbd}`CapsLock` for the Laptop layout, and the {kbd}`Insert` key for the Desktop layout. You can [set the layout](https://canonical-ubuntu-desktop-documentation.readthedocs-hosted.com/en/latest/tutorial/get-started-with-the-screen-reader/#determine-your-keyboard-layout) in the Orca settings, typing {command}`orca -s` in the terminal.
+    
+| Action | Laptop layout | Desktop layout |
+|--------|-----------------------------------|------------------------------------|
+| Enable screen reader | {kbd}`Super+Alt+S` | {kbd}`Super+Alt+S` |
+| Quit screen reader | {kbd}`Super+Alt+S` | {kbd}`Super+Alt+S` |
+| Learn mode | {kbd}`Orca+H` (exit with {kbd}`Esc`) | {kbd}`Insert+H` (exit with {kbd}`Esc`) |
+| Orca preferences | {kbd}`CapsLock+Space` | {kbd}`Insert+Space` |
+| Toggle flat review | {kbd}`CapsLock+P` | {kbd}`Insert+-` on the numeric keypad |
+| Say all (in flat review) | {kbd}`CapsLock` and double-press {kbd}`;` | Double-press {kbd}`+` on the numeric keypad |
+| Read current line | {kbd}`CapsLock+I` | {kbd}`8` on the numeric keypad |
+| Read next line | {kbd}`CapsLock+O` | {kbd}`9` on the numeric keypad |
+| Read previous line | {kbd}`CapsLock+U` | {kbd}`7` on the numeric keypad |
+
+You can find further guidance in the [screen reader documentation](https://canonical-ubuntu-desktop-documentation.readthedocs-hosted.com/en/latest/how-to/accessibility/read-screen-aloud/).
+
+### Screen reading checklist
+
+- All interactive elements (buttons, fields, links…) have a label and a role that is read out loud by the screen reader
+    * Roles explain the function of the component (“button”, “text”, “link”, “combobox”). Check [GTK roles](https://docs.gtk.org/gtk4/enum.AccessibleRole.html).
+- There are no two or more elements with the same label in the same page
```

Comment:
I think this would be easier to parse if it was the other way around:
"All elements on a page have unique labels"

Reply:
Rewrite looks good to me, much clearer :)